src/player/index.ts

```diff
@@ -55,8 +55,7 @@ const TOUR_REFERENCE_PATTERN =
   /(?:\[(?<linkTitle>[^\]]+)\])?\[(?=\s*[^\]\s])(?<tourTitle>[^\]#]+)?(?:#(?<stepNumber>\d+))?\](?!\()/gm;
 const FILE_REFERENCE_PATTERN = /(\!)?(\[[^\]]+\]\()(\.[^\)]+)(?=\))/gm;
 const CODE_FENCE_PATTERN = /```[^\n]+\n(.+)\n```/gms;
-
-const DIVIDER = "\n\n---\n";
+const DIVIDER = "\n";
```

Comment:
I wonder if we could make the divider conditional on whether `isAccessibilitySupportOn()` is true - that way we'd be leaving the default behavior as is, but cleaning things up for the screenreader output?

Reply:
awesome!